frontend/apps/web/app/(mgmt)/[account]/connections/[id]/components/MssqlForm.tsx

```diff
@@ -61,7 +61,7 @@ export default function MssqlForm(props: Props): ReactElement {
   const form = useForm<MssqlFormValues, MssqlEditConnectionFormContext>({
     mode: 'onChange',
     resolver: yupResolver(MssqlFormValues),
-    values: defaultValues,
+    defaultValues: defaultValues,
```

Comment:
This fixes an issue I was seeing where on submit, sometimes values would revert back to their old value until a page refresh.